context/AuthHelper.tsx

```diff
@@ -49,3 +55,30 @@ export const loginAccount = async ({
     return error as Error;
   }
 };
+
+/**
+ * Log out and clear session state
+ * @returns {Promise<void | Error>}
+ */
+export const logoutFunction = async ({
+  setIsSignedIn,
+  router,
+  resetUser,
+}: logoutType) => {
+  try {
+    throw Error('new error');
```

Comment:
This looks like a test code. This needs to be removed or this logout will never work.